src/qibocal/protocols/dispersive_shift.py

```diff
@@ -93,73 +95,56 @@ def _acquisition(
         params (DispersiveShiftParameters): experiment's parameters
         platform (Platform): Qibolab platform object
         targets (list): list of target qubits to perform the action
-
     """
 
-    # create 2 sequences of pulses for the experiment:
-    # sequence_0: I  - MZ
-    # sequence_1: RX - MZ
-
-    # taking advantage of multiplexing, apply the same set of gates to all qubits in parallel
     sequence_0 = PulseSequence()
     sequence_1 = PulseSequence()
-    ro_pulses = {}
-    qd_pulses = {}
     for qubit in targets:
-        qd_pulses[qubit] = platform.create_RX_pulse(qubit, start=0)
-        ro_pulses[qubit] = platform.create_qubit_readout_pulse(
-            qubit, start=qd_pulses[qubit].duration
-        )
-        sequence_0.add(ro_pulses[qubit])
-        sequence_1.add(qd_pulses[qubit])
-        sequence_1.add(ro_pulses[qubit])
+        natives = platform.natives.single_qubit[qubit]
+        sequence_0 |= natives.MZ()
+        sequence_1 |= natives.RX()
+        sequence_1 |= natives.MZ()
```

Comment:
Same issue as the Zeno
```suggestion
        sequence_0 += natives.MZ()
        sequence_1 += (natives.RX() | natives.MZ())
```
if you want to play different qubits in parallel (which I believe is the 0.1 behavior).